crates/client/src/storage/fs.rs

```diff
@@ -120,6 +121,45 @@ impl RegistryStorage for FileSystemPackageStorage {
         Ok(packages)
     }
 
+    async fn load_operator(&self) -> Result<Option<OperatorRecord>> {
```

Comment:
As stated earlier, if this were to be loading an `OperatorInfo`, I don't think the changes to make `OperatorRecord` serializable would be necessary.

Reply:
I'd also expect the implementation to look more like:

```rust
Ok(load(&self.operator_path()).await?)
```